### DISCUSSION TITLE
Building app with Xcode 14.3: GRDBQuery doesn't build 'cuz @StateObject requires macOS 11.0

### DISCUSSION THREAD

**Discussion by crenelle** [groue/GRDB.swift#1356] [General]
I'm using GRDB but not GRDBQuery (as var as I know :) this may send me back to Xcode 14.2 for a while.

**Comment by crenelle** [comment]
Ah, whew! I had included GRDBQuery a while back expecting to use it or at least look at it, but had not. When the build error showed up, I thought that GRDBQuery was part of the GRDB swift package and not a separate package. I just deleted GRDBQuery from the build, it all builds now. A little strange that the GRDBQuery README still showed it was compatico with macOS 10.15.

**Reply by groue** [reply]
A fix was recently pushed: https://github.com/groue/GRDBQuery/pull/37
